### ISSUE TITLE
Improperly HTML escaped bio

### ISSUE THREAD

**Issue by frewsxcv** [OSFeels/OSFeels.com#70]
Double quotes appear within the `title` attribute

```
<div class="speakers--info-box" data-toggle="tooltip" title="Tute Costa studied Computer Sciences in Argentina, worked on and contributed to different Open Source Projects, and now works at thoughtbot NYC. He released a beta version of the book "Maintaining Open Source Projects", where he covers mainly the non-technical aspects of the job.">
```

**Comment by strand** [comment]
Resolved by #71

**Comment by frewsxcv** [comment]
:sparkles:
